metrics-servlets/src/main/java/com/codahale/metrics/servlets/HealthCheckServlet.java

```diff
@@ -124,7 +124,8 @@ protected void doGet(HttpServletRequest req,
         if (results.isEmpty()) {
             resp.setStatus(HttpServletResponse.SC_NOT_IMPLEMENTED);
         } else {
-            if (isAllHealthy(results)) {
+            final boolean alwaysOk = Boolean.parseBoolean(req.getParameter("alwaysOk"));
```

Comment:
Could we rename the parameter to `overrideStatusCode`?

The name `alwaysOk` implies that the health check result would be always healthy. 😉

Reply:
Initially I planned to name it similar to your proposal, but then I thought this would suggest the parameter value should be the desired status code (`overrideStatusCode=200`) instead of a boolean. I included `OK` because this is the official name for HTTP status 200. Also, I think `OK` differs from `healthy` enough to not suggest it would affect those entries.

Maybe `httpStatusOk=true`?